lib/optimizely/odp/odp_config.rb

```diff
@@ -46,7 +46,9 @@ def initialize(api_key = nil, api_host = nil, segments_to_check = [])
     def update(api_key = nil, api_host = nil, segments_to_check = [])
       updated = false
       @mutex.synchronize do
+        initial_state = @odp_state
         @odp_state = api_host.nil? || api_key.nil? ? ODP_CONFIG_STATE[:NOT_INTEGRATED] : ODP_CONFIG_STATE[:INTEGRATED]
+        updated = @odp_state != initial_state
```

Comment:
Not clear why this update again in addition to line 57? Does it make difference?